pycqed/analysis_v2/base_analysis.py

```diff
@@ -199,9 +204,39 @@ def process_data(self):
     def prepare_plots(self):
         pass
 
-    def run_fitting(self):
+    def prepare_fitting(self):
         pass
 
+    def run_fitting(self):
+        # This function does the fitting and saving of the parameters
+        # based on the fit_dict options.
+        # Only model fitting is implemented yet. Minimizing fitting should
+        # be implemented here.
+        fit_guess_fn = self.fit_dict['fit_guess_fn']
```

Comment:
I thought a bit about this and I think `fit_dicts` is the right name for this. 
However, I would make this a dict of dicts for each fit that one wants to do on the dataset. 

For example, in the flipping analysis I'm making now I want to do both a fit to a cosine (or a decaying cosine) and to a polynomial. 

I would like the fit dict to have the following structure. 
```python 
self.fit_dict{'cos_fit:  {
                                    'fit_guess_fn': some_fun, 'fit_xvals': fit_xval}
                    'poly_fit': {'fit_guess_fn': some_other_fun , .... } 
                    } 
```